recipe/meta.yaml

```
@@ -1,42 +1,57 @@
 {% set name = "bidict" %}
-{% set version = "0.21.2" %}
-{% set compress_type = "tar.gz" %}
-{% set hash_val = "4fa46f7ff96dc244abfc437383d987404ae861df797e2fd5b190e233c302be09" %}
+{% set version = "0.23.1" %}
+{% set commit_id = "636f5a610f0e0b9a1e056fc72b688720c114403b" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ compress_type }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ hash_val }}
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 03069d763bc387bbd20e7d49914e75fc4132a41937fa3405417e1a5a2d006d71
+    # tests and tests data files
+  - url: https://github.com/jab/bidict/archive/{{ commit_id }}.tar.gz
```

You can just switch to this source and remove pypi.
```suggestion
  - url: https://github.com/jab/{{ name }}/archive/{{ commit_id }}.tar.gz
```

---

shasum -a 256 bidict-0.23.1.tar.gz 
b3619436e1e1e3cba15856839666edcb769fce97b47f5bba5e2789b03eed3156  bidict-0.23.1.tar.gz